### PR TITLE
add Template to Alcatraz

### PR DESCRIPTION
**Pull request by alexchueh**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -1055,6 +1055,11 @@
         "description": "Bringing the Empty Application template back to Xcode",
         "screenshot": "https://raw.githubusercontent.com/Isuru-Nanayakkara/Xcode-Empty-Application-Template/master/screenshot.png"
       }
```
Comment (jurre) [inline packages.json]: Hey, you're missing a comma here :)

Comment (alexchueh) [inline packages.json]: oh!!Thanks i miss this XD...

File: packages.json [hunk 2]
```diff
@@ -1054,7 +1054,12 @@
         "url": "https://github.com/Isuru-Nanayakkara/Xcode-Empty-Application-Template",
         "description": "Bringing the Empty Application template back to Xcode",
         "screenshot": "https://raw.githubusercontent.com/Isuru-Nanayakkara/Xcode-Empty-Application-Template/master/screenshot.png"
-      }
+      },
+      {
+        "name": "PIXNETTemplate",
+        "url": "https://github.com/pixnet/PIXNETTemplate",
+        "description": "File Template for PIXNET coding",
```
Comment (jurre) [inline packages.json]: And I think this trailing comma is causing the current failure, sorry :P

Comment (alexchueh) [inline packages.json]: ok i'm fix

**Comment by jurre** [comment]
Hey, it was still failing and under the wrong key so I've just added it for you. Thanks!

**Pull request closed without merging**